hack/manifests/csi/vsphere/05-example-pod.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example-windows-pod
+  # must be in the same namespace as the PVC
+  namespace: windows-webserver
+spec:
+  os:
+    name: windows
+  tolerations:
+  - key: "os"
+    value: "Windows"
+    Effect: "NoSchedule"
+  nodeSelector:
+    kubernetes.io/os: windows
+  containers:
+    - name: test-container
```

Comment:
Consider naming the container according to the running example. e.g. example-windows-container